LeaderboardBackend/Program.cs

```diff
@@ -262,6 +262,11 @@
     options.InvalidModelStateResponseFactory = context =>
     {
         ValidationProblemDetails problemDetails = new(context.ModelState);
+        if (problemDetails.Errors.ContainsKey("$"))
+        {
+            return new BadRequestObjectResult(problemDetails);
+        }
```

Comment:
This looks like a hack. Can you put a code comment here explaining what the purpose of this is?

Reply:
Oh yeah this is. I can't figure out a more elegant solution, so I did this.

I'll add a comment here, sure

Reply:
Thank you. This is the kind of thing we're going to come back to in two months and be scratching our heads over.